docs/TROUBLESHOOT.md

```diff
@@ -0,0 +1,74 @@
+### Description
+
+Upon running the <docker-compose -f docker-compose.yml -f docker-compose.override.yml up> all the containers are not started mentioned in the docker-compose file. Some containers throw certain errors mentioned below
+
+### To Start/Run the app
+
+◦	docker-compose up -d
+◦	docker-compose stop ibf-dashboard
```

Comment:
Please add something here, that the bottom 3 lines of these 4 are just a suggestion, with the benefit that changes in front-end code are immediately reflected.